tests/queries/saved_queries/api_tests.py

```diff
@@ -326,17 +326,22 @@ def test_get_saved_query_favorite_filter(self):
         self.login(username="admin")
         uri = f"api/v1/saved_query/?q={prison.dumps(arguments)}"
         rv = self.client.get(uri)
-        self.assertEqual(rv.status_code, 200)
         data = json.loads(rv.data.decode("utf-8"))
-        self.assertEqual(data["count"], len(expected_models))
+        assert rv.status_code == 200
```

Comment:
why not use self.assert200(rv)?

Reply:
Because it's used nowhere on the test codebase, not written in stone but with `pytest` now we are inclined to use `assert`